platforms/platform-alexa/src/AlexaRequest.ts

```diff
@@ -202,6 +202,10 @@ export class AlexaRequest extends JovoRequest {
     return this.context!.System.apiAccessToken;
   }
 
+  getUnit(): Unit|undefined {
```

Comment:
Hey @kouz75 
Thanks again for the PR :)

The last comment may have been lost. Could you please add spaces into the return type. 

`getUnit(): Unit | undefined {` 

Otherwise, the linter will complain :)